### PR TITLE
KFSPTS-18320 Backport FINP-6611

### PR DESCRIPTION
**Pull request by jhulslander**
This commit brings in InvoiceGeneralDetail into our overlay.  The only edit to the class is where kualiModuleService is defined, and the comments above it.

### CODE REVIEW

File: src/main/java/org/kuali/kfs/module/ar/businessobject/InvoiceGeneralDetail.java [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    // To categorize the CG Invoices based on Award LOC Type
+    private String letterOfCreditCreationType;
+    private String letterOfCreditFundGroupCode;
+    private String letterOfCreditFundCode;
+
+    private ContractsGrantsInvoiceDocument invoiceDocument;
+    private ContractsAndGrantsBillingAward award;
+    
+    /*
+     *  brought in InvoiceGeneralDetail as a new overlay class so wan backport FINP-6611
```
Comment (ajd299) [inline src/main/java/org/kuali/kfs/module/ar/businessobject/InvoiceGeneralDetail.java:67]: typo found in comments: "wan"

File: src/main/java/org/kuali/kfs/module/ar/businessobject/InvoiceGeneralDetail.java [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+     *  base code as of our current overlay kualiModuleService is not a transient service. FINP-6611 makes it transient
+     *  This overlay should be removed when the 3/26 version of KFS is brought in
+     */
+    private transient KualiModuleService kualiModuleService;
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/org/kuali/kfs/module/ar/businessobject/InvoiceGeneralDetail.java:71]: yes, just this line, added transient

File: src/main/java/org/kuali/kfs/module/ar/businessobject/InvoiceGeneralDetail.java (hunk 1 above)
Comment (jhulslander) [reply src/main/java/org/kuali/kfs/module/ar/businessobject/InvoiceGeneralDetail.java:67]: OK, will fix

**Comment by jhulslander** [comment]
I fixed the comment, this is ready for review again

**Pull request merged**